### PR TITLE
Add style on the contact form

### PR DESCRIPTION
**Pull request by haroldmud**
**In this pull request, the following requirements were implemented:**
 - My Partner and I : 
   - resized the contact form title and icons
   - adjust the text block
   - add a box-shadow hover effect on the submission button

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -179,5 +179,43 @@ <h2 class="title-2">ROAD TRIP PLANNER</h2>
...
+                        <input type="text" name="user_full_name" id="full-name" placeholder="Text Input" required="" maxlength="30">
+                        <label for="email-address" hidden="">Email Address</label>
+                        <input type="email" name="user_email" id="email-address" placeholder="jeremy@gmail.com" required="">
+                        <label for="message" hidden="">Message</label>
+                        <textarea rows="5" name="user_message" id="message" placeholder="Enter text here" required="" maxlength="500"></textarea>
+                    </fieldset>
+                    <div class="container-5">
+                        <button class="btn-1">See Project</button>
+                    </div>
+                </form>
```
Comment (ShadyShawkat) [inline index.html]: - [ ] Kindly include the 'See Project' button in the form. This button is supposed to be the submit button of this form.

File: index.html [hunk 2]
```diff
@@ -179,5 +179,43 @@ <h2 class="title-2">ROAD TRIP PLANNER</h2>
             </section>
         </section>
     </div>
+    <footer class="footer">
+        <section class="contact-1">
+                <h1 class="title-1">
+                    CONTACT US
+                </h1>
```
Comment (ShadyShawkat) [inline index.html]: - [ ] Please change this `h1` tag to any other heading tag `t2`-`t6`. Your web page should contain only one `h1` tag for the main title of the page and then other heading tags depending on the hierarchy of the headings.

**Review by ShadyShawkat — CHANGES_REQUESTED** [review]
## Hi @haroldmud, 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!

Image

## Required Changes ♻️
- [ ] Kindly edit your readme to look more professional, you can use git syntax for styling which is used in [this](https://github.com/microverseinc/readme-template) readme template provided by Microverse. (You can see its 'raw' text to see how the text is displayed like [this](https://raw.githubusercontent.com/microverseinc/readme-template/master/README.md))
- [ ] _Check the comments under the review for more changes._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by prolajumokeoni — APPROVED** [review]
Hi @braincee  and @haroldmud, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by braincee** [comment]
Thanks very much, @haroldmud, @ShadyShawkat and @prolajumokeoni 

**_Note: Please I have implemented all the required and optional changes._**

tantan-lamronspace

**Pull request merged**